src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -352,6 +352,10 @@ var ReactDOM = {
   },
 
   unmountComponentAtNode(container : DOMContainerElement) {
+    invariant(
+      isValidContainer(container),
+      'unmountComponentAtNode(...): Target container is not a DOM element.'
+    );t
```

Comment:
@bvaughn but `renderSubtreeIntoContainer` is only called if `container._reactRootContainer` exists, so it wouldn't actually throw, it would just be a no-op.